bash3/arguments.sh

```diff
@@ -0,0 +1,12 @@
+# Script to read an argument and print as lowercase in stdout
+
+#!/bin/bash
```

Comment:
[Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) should always be the first line of the file.

Reply:
And fix for all the others, too